### PR TITLE
Validate ClinVar submission using the dry-run endpoint

### LINKED ISSUE #3795: Avoid using ClinVar apitest from Scout prod - redirect data from scout-stage to apitest
That corresponds to the preClinVar `dry_run` endpoint, not the `validate` one. 

They look the same to me but on their docs they say that the validate one should be used only for building the pipeline, not for testing each single submission.

### PR DESCRIPTION
**Pull request by northwestwitch**
Fixes partially #3795 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. Try to validate a ClinVar submission object from Scout stage

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [ ] tests executed by

### CODE REVIEW

File: scout/server/extensions/clinvar_extension.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ class ClinVarApi:
 
     def __init__(self):
         self.convert_service = "/".join([PRECLINVAR_URL, "csv_2_json"])
-        self.validate_service = "/".join([PRECLINVAR_URL, "validate"])
+        self.validate_service = "/".join([PRECLINVAR_URL, "dry-run"])
```
Comment (dnil) [inline scout/server/extensions/clinvar_extension.py:20]: Good! Does this work as expected now with actually retrieving a new SUB-number?

Comment (northwestwitch) [reply scout/server/extensions/clinvar_extension.py:20]: It should, but let's try it!

File: .wokeignore [hunk 2]
```diff
@@ -1 +1,2 @@
 scout/server/blueprints/login/static/aspirant.svg
+scout/demo/rnafusion_report_example.html
```
Comment (northwestwitch) [inline .wokeignore:2]: This keeps triggering the woke check. I guess it would be best to fix it, but not here, since it's created by an external software.

**Review by dnil — APPROVED** [review]
Looks good to me! I still feel we have a couple of things here to fix, with staging and the maybe the return of SUB-ids isn't quite doing what we think it does. But this PR is conforming better to what they have in their docs, so all for!

**Comment by northwestwitch** [comment]
Tested on stage. 
- When clicking on `Validate`: 🆗 

image

- When clicking on Validate and obtain ID from ClinVar it's not returning an ID: ⛔ 

image

This needs fixing!

**Comment by northwestwitch** [comment]
Ok, I start to think that we can't use the dry-run as we used the validate endpoint. 

Checking at [my code on preClinVar](https://github.com/Clinical-Genomics/preClinVar/blob/87a1efee64495d8730392b2f8b23d8cc590c0ec7/preClinVar/main.py#L71)

And what's reported on the ClinVar API:

image

Now I see that a successful response has code 204 and is EMPTY, meaning that we can't retrieve a new submission name (SUBXXX) using that endpoint. 

I think we have to stick with the current code!

**Comment by northwestwitch** [comment]
I'm going to close this PR now!

**Pull request closed without merging**